ui/package.json

```diff
@@ -138,16 +143,16 @@
     "@types/google.analytics": "0.0.39",
     "@types/jasmine": "^3.3.5",
     "@types/jasminewd2": "^2.0.3",
-    "@types/jest": "^23.3.11",
+    "@types/jest": "^26.0.21",
     "@types/node": "^10.12.11",
     "@types/react-router-dom": "^5.1.5",
     "@typescript-eslint/eslint-plugin": "^4.15.2",
     "@typescript-eslint/parser": "^4.15.2",
     "codelyzer": "^4.1.0",
     "enzyme": "^3.11.0",
-    "enzyme-adapter-react-16": "^1.15.5",
+    "enzyme-adapter-react-16": "^1.15.6",
```

Comment:
I thought test failure was a enzyme issue. but upgrade to latest version didn't help.